bin/cron/geocode_hoc_activity

```diff
@@ -31,14 +31,18 @@ def main
       if location.latitude && location.longitude
         position = "#{location.latitude}, #{location.longitude}"
       end
-      DB[:hoc_activity].where(id: row[:id]).update(
-        country: location.country,
-        country_code: location.country_code,
-        state: location.state,
-        state_code: location.state_code,
-        city: location.city,
-        location: position,
-      )
+      if location.state_code.length > 2
```

Comment:
could the `state_code` attribute be `nil`?  Maybe `location.state_code&.length` is safer?